cloud_AWS/terraform/module/cloudexport.tf

```diff
@@ -7,22 +7,22 @@ terraform {
 }
 
 resource "kentik-cloudexport_item" "aws_export" {
-  # Create only when plan_id is set
-  count = var.plan_id == "" ? 0 : 1
+  # Create when bucket_name_list or plan_id is set
+  count = var.bucket_name_list != null ? length(var.bucket_name_list) : var.plan_id == "" ? 0 : 1
 
-  name           = "${var.name}-${terraform.workspace}" # cloudexport name must be unique
+  name           = var.bucket_name_list != null ? "${var.name}-${terraform.workspace}${count.index}" : "${var.name}-${terraform.workspace}" # cloudexport name must be unique
   type           = "CLOUD_EXPORT_TYPE_KENTIK_MANAGED"
   enabled        = var.enabled
   description    = var.description
   plan_id        = var.plan_id
   cloud_provider = "aws"
   aws {
-    bucket = join(", ", [
+    bucket = var.bucket_name_list != null ? "${var.bucket_name_list[count.index]}/" : join(", ",[
```

Comment:
I think the slash / included in "bucket" name is invalid

Reply:
Kentik portal reports "The S3 Bucket Name attribute has errors." with that slash

Reply:
Resolved